app/services/albums.js

```diff
@@ -0,0 +1,15 @@
+const request = require('request-promise');
+
+exports.getAlbumsApi = () =>
+  request('https://jsonplaceholder.typicode.com/albums')
```

Comment:
here are several comments:
- what happen if the user wants a specific album?
- if the album's url endpoint changes, you will have to come to the code and change it in each occurrence i think that it must be an environment variable that you can expose by config files.

Reply:
Thanks for the comment.